README.md

```diff
@@ -19,13 +19,14 @@ has an `image` field which will be used to select the matching containers in the
 
 Signature types:
 
-1. GitHub actions. It will verify that all images were signed for a GitHub action with the `kubewarden` owner and in the repo `app-example`
+1. GitHub actions. It will verify that all images were signed for a GitHub action with the `kubewarden` owner and in the repo `app-example`.
```

Comment:
I know this is a PITA, but please propagate all these changes also to the `metadata.yml` file

Reply:
argh, true. Fixed in https://github.com/kubewarden/verify-image-signatures/pull/25/commits/eb859b5083af811c3529b2559eea461615e794b6. I look forward to a make target, once we do the artifacthub metadata.